python/helpers/requirements.txt

```diff
@@ -1,5 +1,5 @@
-pip==22.0.4
-pip-tools==6.5.1
+pip>=21.3.1
+pip-tools>=6.4.0
```

Comment:
When `pip install -r` runs won't `>=` just pick the latest version and not restore Python 3.6 support?

If so, I think we should pin it with `==` and tell dependabot `@dependabot ignore this minor version` if it tries to bump it. 

When `6.6.0` releases, dependabot will make a PR and we can re-evaluate whether to update to `6.5.X`. 

Maybe also add a comment here as to why we're lagging behind on these releases, so when dependabot makes a PR we don't just merge it right on in!

Does that sound like it would work? 🤔 Do we have a policy on what older versions we support?

Reply:
I tested this setup on ubuntu 18.04 (python 3.6.9) and it does the correct thing.  Pip detects that the latest "compatible" version is 21.3.1 and 6.4.0 and correctly installs those on python 3.6, also correctly installs the actual latest version on python >= 3.7.

Comments incoming :)